docs/mongocli/command/mongocli-atlas.txt

```diff
@@ -89,6 +90,7 @@ Related Commands
    dbusers </command/mongocli-atlas-dbusers>
    events </command/mongocli-atlas-events>
    integrations </command/mongocli-atlas-integrations>
+   kubernetes </command/mongocli-atlas-kubernetes>
```

Comment:
can you please make sure this command is not added to `mongocli`, atlas command under mongocli are deprecated and flagged to be removed we don't want to add new features there unless really necessary

Reply:
Got it, thanks! I'll remove it from `mongocli`